kubeportal/models/news.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.core.validators import MaxValueValidator, MinValueValidator
+from tinymce.models import HTMLField
+
+from . import User
+
+class News(models.Model):
+    """
+    News on the portal front page.
+    """
+    title = models.CharField(null=False, max_length=40)
+    content = HTMLField()
```

Comment:
I think using a HTMLField is taking a risk, because we can't restrict what the user enters and it may or may not break the frontend HTML.

Reply:
HTML input is restricted by the TinyMCE editor in the admin backend. I really want to have support for links in the news section, so I vote for keeping the HTML field and restrict it on the editor side.

Reply:
approved